apps/site/pages/en/blog/release/v24.0.0.md

```diff
@@ -0,0 +1,490 @@
+- \[[`7e3503fff1`](https://github.com/nodejs/node/commit/7e3503fff1)] - **util**: fix parseEnv handling of invalid lines (Augustin Mauroy) [#57798](https://github.com/nodejs/node/pull/57798)
+- \[[`594269fcca`](https://github.com/nodejs/node/commit/594269fcca)] - **util**: fix formatting of objects with built-in Symbol.toPrimitive (Shima Ryuhei) [#57832](https://github.com/nodejs/node/pull/57832)
+- \[[`8ca56a8db8`](https://github.com/nodejs/node/commit/8ca56a8db8)] - **util**: preserve `length` of deprecated functions (Livia Medeiros) [#57806](https://github.com/nodejs/node/pull/57806)
+- \[[`6add4c56aa`](https://github.com/nodejs/node/commit/6add4c56aa)] - **util**: fix parseEnv incorrectly splitting multiple ‘=‘ in value (HEESEUNG) [#57421](https://github.com/nodejs/node/pull/57421)
+- \[[`e577618227`](https://github.com/nodejs/node/commit/e577618227)] - **util**: inspect: enumerable Symbols no longer have square brackets (Jordan Harband) [#55778](https://github.com/nodejs/node/pull/55778)
+- \[[`cb7eb15161`](https://github.com/nodejs/node/commit/cb7eb15161)] - **watch**: clarify completion/failure watch mode messages (Dario Piotrowicz) [#57926](https://github.com/nodejs/node/pull/57926)
+- \[[`65562127bd`](https://github.com/nodejs/node/commit/65562127bd)] - **watch**: check parent and child path properly (Jason Zhang) [#57425](https://github.com/nodejs/node/pull/57425)
+- \[[`b39fb9aa7f`](https://github.com/nodejs/node/commit/b39fb9aa7f)] - **win**: fix SIGQUIT on ClangCL (Stefan Stojanovic) [#57659](https://github.com/nodejs/node/pull/57659)
+- \[[`76c5ea669d`](https://github.com/nodejs/node/commit/76c5ea669d)] - **worker**: add ESM version examples to worker docs (fisker Cheung) [#57645](https://github.com/nodejs/node/pull/57645)
+- \[[`17965eb33d`](https://github.com/nodejs/node/commit/17965eb33d)] - **zlib**: fix pointer alignment (jhofstee) [#57727](https://github.com/nodejs/node/pull/57727)
+
+Windows 64-bit Installer: https://nodejs.org/dist/v24.0.0/node-v24.0.0-x64.msi \
+Windows ARM 64-bit Installer: https://nodejs.org/dist/v24.0.0/node-v24.0.0-arm64.msi \
+Windows 64-bit Binary: https://nodejs.org/dist/v24.0.0/win-x64/node.exe \
+Windows ARM 64-bit Binary: https://nodejs.org/dist/v24.0.0/win-arm64/node.exe \
+macOS 64-bit Installer: https://nodejs.org/dist/v24.0.0/node-v24.0.0.pkg \
+macOS Apple Silicon 64-bit Binary: https://nodejs.org/dist/v24.0.0/node-v24.0.0-darwin-arm64.tar.gz \
+macOS Intel 64-bit Binary: https://nodejs.org/dist/v24.0.0/node-v24.0.0-darwin-x64.tar.gz \
+Linux 64-bit Binary: https://nodejs.org/dist/v24.0.0/node-v24.0.0-linux-x64.tar.xz \
+Linux PPC LE 64-bit Binary: https://nodejs.org/dist/v24.0.0/node-v24.0.0-linux-ppc64le.tar.xz \
+Linux s390x 64-bit Binary: https://nodejs.org/dist/v24.0.0/node-v24.0.0-linux-s390x.tar.xz \
+AIX 64-bit Binary: https://nodejs.org/dist/v24.0.0/node-v24.0.0-aix-ppc64.tar.gz \
+ARMv7 32-bit Binary: _Coming soon_ \
```

Comment:
This is not coming anytime soon, is it?
```suggestion
```

Reply:
We'll need to tweak the blog generator otherwise this will keep reappearing for every new 24.x release blog post.